<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hopDock</class>
 <widget class="QDockWidget" name="hopDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>257</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>257</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="../brewtarget.qrc">
    <normaloff>:/images/smallHop.svg</normaloff>:/images/smallHop.svg</iconset>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Hop &amp;dock</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Beta (%)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_beta</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="BtGenericEdit" name="lineEdit_beta">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">beta_pct</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>HSI</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_HSI</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="BtGenericEdit" name="lineEdit_HSI">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">hsi_pct</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Origin</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_origin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="BtStringEdit" name="lineEdit_origin">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">origin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Humulene (%)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_humulene</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="BtGenericEdit" name="lineEdit_humulene">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">humulene_pct</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Caryophyllene (%)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_caryophyllene</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="BtGenericEdit" name="lineEdit_caryophyllene">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">caryophyllene_pct</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Myrcene (%)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_myrcene</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="BtGenericEdit" name="lineEdit_myrcene">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">myrcene_pct</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="BtStringEdit" name="lineEdit_name">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Cohumulone (%)</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_cohumulone</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="BtGenericEdit" name="lineEdit_cohumulone">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="editField" stdset="0">
         <string notr="true">cohumulone_pct</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtGenericEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtStringEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../brewtarget.qrc"/>
 </resources>
 <connections/>
</ui>
